actix-multipart/src/form/bytes.rs

```diff
@@ -0,0 +1,45 @@
+//! Reads a field into memory.
+use crate::form::{FieldReader, Limits};
+use crate::{Field, MultipartError};
+use actix_web::HttpRequest;
+use bytes::{Bytes, BytesMut};
+use futures_core::future::LocalBoxFuture;
+use futures_util::TryStreamExt;
+use mime::Mime;
+
+/// Read the field into memory.
+#[derive(Debug)]
+pub struct BytesField {
```

Comment:
ik we talked about this earlier, but maybe we can come to a compromise lol--what do you think about removing the `Field` suffix on all these (`BytesField`, `JsonField`, etc)? there's some naming collisions, which is annoying on the library side for sure. maybe split these into a module `field` for clarity so users will have `field::Bytes` as an option instead, and the library can deal with reexporting?

just feels bad as a user to type the same thing over and over, and the purpose of the wrappers is pretty clear in a user's code when you see them in a `Multipart` struct, so I don't think specifying `Field` gives that much more information

Reply:
Yeah that sounds like a good idea - and I'm definitely happy to reconsider the names of basically every type in this PR 😆 

One thing I am worried about is a repeat of this issue: https://github.com/jacob-pro/actix-easy-multipart/issues/2#issuecomment-1225384360

Which could easily happen since we would have both a multipart `JsonConfig` and a regular actix-web `JsonConfig` structs - which might get very confusing...

Reply:
there's definitely room for confusion, but I think it's mostly solved by more detailed documentation. plus the situation here is mitigated by the fact that the name conflicts are in different crates-- much easier to spot the difference between `actix_web::web::Json` and `actix_web_multipart::field::Json` (or whatever the path might be) than two types in `actix_web_multipart`

would also be nice to shorten the attribute macro `multipart_form` -> `multipart`-- matches the crate name, which is at least what I would expect.

Reply:
Makes sense - I have renamed them all as described

Reply:
Regarding the macro name (and also extractor / trait name) - I was trying to be careful to keep it separate from the existing low-level implementation.

People may still want to use the `Multipart` extractor which just provides access to stream itself, or they can use the higher-level `MultipartForm` extractor which allows automatic extraction of typed fields.

Reply:
makes sense, I was just referring to the attribute macro `#[multipart_form]`, by analogy with `serde`-- i.e. using `#[serde]` as opposed to `#[deserialize]`, and so `#[multipart]` instead of `#[multipart_form]`. I don't think the lower level `Multipart` has a proc macro/attributes, but if it does it certainly makes sense to keep those separate.

Reply:
oh I see what you mean - yep that is a good idea - done 👍